src/js/modules/add-to-watched&queue.js

```diff
@@ -0,0 +1,39 @@
+import Notiflix from 'notiflix';
+import { watched, saveToWatchedLocalStorage } from './fetchWatched.js';
+import { queue, setQueueLocalStoradge } from './fetchQueued.js';
+
+export function onAddToWatched(id) {
+  if (watched.includes(id)) {
+    Notiflix.Notify.warning('This movie is already in your watched list.');
+    return;
```

Comment:
Ten sposób sprawdzania może się nie sprawdzić kiedy np. w zbiorze będzie id 1234, wtedy np. 123 da już matcha. Naprawię to potem sam, lub jeśli chcesz, można dopracować.